lmod/SitePackage_licenses.lua

```diff
@@ -132,6 +132,18 @@ function find_and_define_license_file(environment_variable,application)
 			license_found = true
 		end
 	end
+	
+	-- Fifth, look at the restricted repository for a file called by the cluster's name with priority
+	local dir = pathJoin("/cvmfs/soft.computecanada.ca/config/licenses/",application,"clusters")
```

Comment:
shouldn't this line read:
```lua
local dir = pathJoin("/cvmfs/restricted.computecanada.ca/config/licenses/",application,"clusters")
```
i.e.  `s/soft.computecanada.ca/restricted.computecanada.ca/`

Reply:
Yes Maxime corrected that here: https://github.com/ComputeCanada/software-stack-config/commit/7f6a970ec50036035d9fdf147bbc8d32a321600a